apps/dashboard/config/configuration_singleton.rb

```diff
@@ -47,6 +47,7 @@ def boolean_configs
       :host_based_profiles          => false,
       :disable_bc_shell             => false,
       :cancel_session_enabled       => false,
+      :hide_app_version             => false,
```

Comment:
Tests are failing because there's no entry for this config in the fixture files.

You can see a similar pull request here that added a new config (and the test fixture file I also had to modify)
https://github.com/OSC/ondemand/pull/2398/files